content/source/docs/enterprise/api/run.html.md

```diff
@@ -267,4 +267,12 @@ curl \
   --request POST \
   --data @payload.json \
   https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/discard
-```
\ No newline at end of file
+```
+
+## Available Related Resources
+
+- `plan` - Additional information about plans events.
+- `apply` - Additional information about apply events.
```

Comment:
Here I'd go with either "plan events" and "apply events" or just "plans" and "applies".

Reply:
Sorry, I'm confused on this point. The second suggestion I think would read:

> - `apply` - Additional information about applies.

Would your first suggestion not match what I have now? Maybe you're suggesting simply:

>- `apply` - Apply events.

Reply:
Ah sorry - I meant to call out that it says `plans events`, primarily, but also suggest saying `Additional information about plans` vs `plan events`, for simplicity.